data/inventories/features.csv

```diff
@@ -1,127 +1,127 @@
-N-3,Dependent clause predicate,Оформление сказуемого в придаточном предложении,,,,
-N-4,Compound sentence types,Тип построения сложного предложения,,,,
-N-5,Compound sentence syndesis and asyndeton,Тип связи между элементами сложного предложения,,,,
\ No newline at end of file
+id,en,ru,description_formatted_en,description_formatted_ru,is_multiselect,not_applicable_if,schema_sections
+A-1,Number of degrees of vowel height,Количество степеней подъема гласных,,,,,
+A-2,Vowel height degrees,Подъемы гласных,,,,,
+A-210,Opposition in openness/closedness for vowel height degrees,Противопоставление по открытости/закрытости в подъемах гласных,,,,,
+A-3,Degrees of vowel backness,Ряды гласных,,,,,
+A-4,Number of length degrees,Фонологические ступени долготы,,,,,
+A-5,Vowel opposition in labialization,Противопоставление гласных по лабиализации,,,,,
+A-6,Vowel opposition in nasalization,Противопоставление гласных по назализации,,,,,
+A-7,Vowel opposition in pharyngealization,Противопоставление гласных по фарингализации,,,,,
+A-8,Vowel opposition in ATR,Противопоставление гласных по продвинутости корня языка,,,,,
+A-9,Diphthongs and triphthongs,Наличие дифтонгов и трифтонгов,,,,,
+A-10,Types of diphthongs,Типы дифтонгов,,,,A-9-1,
+A-11,Inventory of obstruents by laryngeal features,Инвентарь шумных согласных по ларингальным признакам,,,,,
+A-12,Inventory of obstruents by manner of articulation,Инвентарь шумных согласных по способу артикуляции,,,,,
+A-13,Inventory of obstruents by place of articulation,Инвентарь шумных согласных по месту образования,,,,,
+A-14,Inventory of labial obstruents by place of articulation,Инвентарь шумных губных согласных по месту образования,,,,,
+A-15,Inventory of coronal obstruents by place of articulation,Шумные переднеязычные согласные по месту образования,,,,,
+A-16,Inventory of guttural obstruents by place of articulation,Шумные заднеязычные согласные по месту образования,,,,,
+A-17,Inventory of postuvular obstruents by place of articulation,Шумные постувулярные согласные по месту образования,,,,,
+A-18,Additional articulatory oppositions of obstruents,Дополнительные артикуляционные противопоставления шумных согласных,,,,,
+A-19,Inventory of sonorants by manner of articulation,Инвентарь сонорных согласных по способу артикуляции,,,,,
+A-20,Inventory of sonorants by place of articulation,Инвентарь сонорных согласных по месту образования,,,,,
+A-21,Additional articulatory oppositions of sonorants,Дополнительные противопоставления сонорных согласных,,,,,
+A-220,Inventory of fricatives by place of articulation,Инвентарь фрикативных согласных по месту образования,,,,,
+B-1,Stress character,Тип ударения,,,,,
+B-2,Stress type,Вид ударения,,,,,
+B-3,Stress carrier,Носитель ударения,,,,B-1-1,
+B-4,Stress fixedness,Фиксированность ударения,,,,B-1-1,
+B-5,Fixed stress carrier,Носитель фиксированного ударения в слове,,,,"B-1-1, B-4-2",

```

Comment:
```suggestion
B-5,Fixed stress carrier,Носитель фиксированного ударения в слове,,,,B-1-1|B-4-2,

```

Reply:
I think it's better to follow the pattern we already have in e.g. [list of doculects](https://raw.githubusercontent.com/Jazyki-Mira/langworld_db_data/feature/add-column-to-features-208/data/inventories/doculects.csv)